actor/builtin/miner/miner_test.go

```diff
@@ -533,7 +533,18 @@ func (mal *minerActorLiason) assertPoStFail(blockHeight uint64, done types.IntSe
 	assert.Equal(mal.t, exitCode, res.Receipt.ExitCode)
 }
 
-func newMinerActorLiason(t *testing.T, st state.Tree, vms vm.StorageMap, ancestors []types.TipSet, minerAddr address.Address ) *minerActorLiason {
+func (mal *minerActorLiason) assertPoStStateAtHeight(expected int64, queryHeight uint64) {
```

Comment:
I'm glad someone else is using the liason!  If you can think of a better name don't hesitate to rename it.